### ISSUE TITLE
0.9.0 fails to build with Carthage

### ISSUE THREAD

**Issue by js** [apollographql/apollo-ios#300]
When using carthage it fails when building the StarwarsAPI scheme:

```
[...]
/Users/johan/Library/Caches/org.carthage.CarthageKit/DerivedData/10.0_10L177m/apollo-ios/0.9.0/Build/Intermediates.noindex/ArchiveIntermediates/StarWarsAPI/IntermediateBuildFilesPath/Apollo.build/Release-iphoneos/StarWarsAPI.build/Script-9FCE2D061E6C251100E34457.sh
/Users/johan/Library/Caches/org.carthage.CarthageKit/DerivedData/10.0_10L177m/apollo-ios/0.9.0/Build/Intermediates.noindex/ArchiveIntermediates/StarWarsAPI/IntermediateBuildFilesPath/Apollo.build/Release-iphoneos/StarWarsAPI.build/Script-9FCE2D061E6C251100E34457.sh: line 10: generate: command not found
Command PhaseScriptExecution failed with a nonzero exit code

** ARCHIVE FAILED **


The following build commands failed:
	PhaseScriptExecution Generate\ Apollo\ Client\ API /Users/johan/Library/Caches/org.carthage.CarthageKit/DerivedData/10.0_10L177m/apollo-ios/0.9.0/Build/Intermediates.noindex/ArchiveIntermediates/StarWarsAPI/IntermediateBuildFilesPath/Apollo.build/Release-iphoneos/StarWarsAPI.build/Script-9FCE2D061E6C251100E34457.sh
(1 failure)
```

Same thing when trying to archive the StarwarsAPI scheme in Xcode. Unsure why it fails to find the `check-and-run-apollo-codegen.sh` during archive

(I _hate_ how Carthage insists on building all the schemes)

**Comment by gbrixey** [comment]
I'm getting this error too.

**Comment by AndrewGold** [comment]
I am seeing this error as well.

**Comment by rushilshah** [comment]
Same as well. Is there any workaround not to build all schemes and just platform specific one and get around with this issue?

**Comment by AndrewGold** [comment]
I was able to work around this by manually removing the StarWarsAPI scheme and running `carthage build --platform iOS`

**Comment by FluffyJack** [comment]
+1

**Comment by tonyarnold** [comment]
It looks like the non-iOS targets are relying on the iOS version of the SQLite framework.

**Comment by nanoxd** [comment]
@tormic I do the following on CI to get past it:

```bash
cd Carthage/Checkouts/apollo-ios/Apollo.xcodeproj/xcshareddata/xcschemes && find . -name "StarWarsAPI.xcscheme" -delete
```

You could also get away with running

```bash
cd Carthage/Checkouts/apollo-ios/Apollo.xcodeproj/xcshareddata/xcschemes && rm StarWarsAPI.xcscheme
```

**Comment by brennantaylor** [comment]
One of these two PRs should resolve the issue:

https://github.com/apollographql/apollo-ios/pull/320
https://github.com/apollographql/apollo-ios/pull/321

**Comment by martijnwalraven** [comment]
Thanks to @brennantaylor, this should be fixed in 0.9.1.
